scripts/exdump_monitor.sh

```diff
@@ -497,7 +497,7 @@ DTIM_latest_geoimr=-0.01
 DTIM_earliest_ascatt=-1.50
 DTIM_latest_ascatt=-0.51
 
-$ushscript_dump/bufr_dump_obs.sh $dumptime 0.50 1 aircar aircft proflr vadwnd \
+$USHbufr_dump/bufr_dump_obs.sh $dumptime 0.50 1 aircar aircft proflr vadwnd \
```

Comment:
Revert to using original variable, $ushscript_dump. It is already defined in JOBSPROC_DUMP_MONITOR.